### ISSUE TITLE
Update session timeout guidance page with feedback from accessibility team

### ISSUE THREAD

**Issue by HelenOsg** [dwp/design-system#124]
## What
Update the guidance on the session timeout pattern following review of content and feedback from Craig

## Why
To implement recommendations

## Done when

-        The standard is WCAG 2.1 not WCAG 2.0.
-        It’s called ‘service timeout’ in the h1 but referred to as ‘session timeout pattern’ in the last paragraph of how it works. Probably should be consistent.
-        There is a typo under when the user is signed in which reads your will be signed out soon instead of you will be signed out soon.
-        Is the text ‘your application will be closed soon’ too specific? We have services where you aren’t applying for something, such as registering a death?
-        There is a typo of w0 minutes on the application has been closed page.

## Related stories

https://github.com/dwp/design-system/issues/48
